vcl_templates/www.vcl.erb

```diff
@@ -250,7 +250,28 @@ sub vcl_recv {
       set req.http.GOVUK-ABTest-EducationNavigation = "A";
     }
   }
-  
+
+  <% ['SearchMatchLength'].each do |test_name| %>
+  if (req.http.User-Agent ~ "^GOV\.UK Crawler Worker") {
```

Comment:
If this is intended for reuse (we'd think we might like to use it), then leaving the crawler worker stuff in would be good for us.